css/properties/max-height.json

```diff
@@ -162,6 +207,70 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "fit-content": {
+            "__compat": {
+              "description": "<code>fit-content</code>",
+              "support": {
+                "chrome": [
+                  {
+                    "version_added": "46"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "22"
+                  }
+                ],
+                "chrome_android": {
+                  "version_added": "46"
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "44"
+                },
+                "opera_android": {
+                  "version_added": "44"
+                },
+                "safari": [
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "6.1"
+                  },
+                  {
+                    "alternative_name": "intrinsic",
+                    "version_added": "2"
+                  }
+                ],
+                "safari_ios": {
+                  "version_added": null
+                },
+                "samsunginternet_android": {
+                  "version_added": "5.0"
+                },
+                "webview_android": {
+                  "version_added": "46"
```

Comment:
Webkit prefix should be true.

Reply:
I'm afraid that I don't understand this comment as we don't use webkit: true anywhere, same as the other similar ones.

Reply:
I made a mistake here. Deleting the comment.

Reply:
Rachel,

Sorry the confusion.

Webview became a Chromium fork starting with version 37. If for a particular feature, Chrome is 37 or greater, webview data should match it exactly. Because the webkit prefix for this feature has never been removed, there needs to be a webkit entry for webview. 

For items that landed in Chromium before 37, webview get's complicated. In the context of this database, webview is [defined to include non-Chromium webviews](https://github.com/mdn/browser-compat-data/blob/master/browsers/webview_android.json). But I don't have access to anything that tells me whether those non-Chromium webviews supported a particular property, much less what version it was in. 

With this particular property, I know that it was in Chromium webview in 37. Since I do not know whether this was in any other webview, `true` is the only thing that safely covers both cases.

Unless I missed something, this should cover all my webview comments. Ping me if I missed something.

Reply:
I'm afraid I still don't understand how this should look. I understand the engine history, I don't understand what change you want made to this PR.

Reply:
It needs to look like this:
```
"webview_android": [
  {
    "version_added": "46"
  },
  {
    "version_added":"true",
    "prefix": "-webkit-:
  }
}
```

Reply:
thank you.